documentation/decisions/0005-supabase-generated-types-implementation.md

```diff
@@ -0,0 +1,75 @@
+# 5. SupaBase-generated-types-implementation
```

Comment:
I don't think it's `SupaBase` but just `Supabase`. Either way not really a request for change or something, it doesn't matter much 😄